module/bdev/ocf/ctx.c

```diff
@@ -777,6 +783,11 @@ vbdev_ocf_ctx_init(void)
 		return ret;
 	}
 
+	ocf_data_allocator = env_allocator_create(PAGE_SIZE, "ocf_data");
```

Comment:
Shouldn't the allocator be deinitialized on ocf ctx cleanup?

Reply:
fixed!